course_catalog/etl/loaders.py

```diff
@@ -589,3 +591,87 @@ def load_content_files(course_run, content_files_json):
         else:
             search_task_helpers.delete_run_content_files(course_run.id)
         return content_files
+
+
+def load_podcast_episode(episode_data, podcast):
+    """Load a podcast_episode into the database"""
```

Comment:
Can we add arguments and a return value for this docstring?

Reply:
I think you missed this comment

Reply:
Fixing!